packages/compartment-mapper/src/parse-cjs-shared-export-wrapper.js

```diff
@@ -176,10 +181,14 @@ export const wrap = ({
     if (exportsHaveBeenOverwritten) {
       moduleEnvironmentRecord.default = finalExports;
       for (const prop of keys(moduleEnvironmentRecord.default || {})) {
-        if (prop !== 'default')
+        if (prop !== 'default') {
           moduleEnvironmentRecord[prop] = moduleEnvironmentRecord.default[prop];
+        }
       }
     }
+    for (const prop of redefined) {
+      moduleEnvironmentRecord[prop] = moduleEnvironmentRecord.default[prop];
```

Comment:
Took me a while to get to this conclusion, but the only way this covers anything that was not covered before in the for loop above is if a prop is not enumerable. Otherwise it's already been looped through in the same synchronous run.
How is this changing anything?

(yes, I'm gonna go run the tests now)